rocky/reports/views/view_helpers.py

```diff
@@ -0,0 +1,84 @@
+from django.urls import reverse_lazy
+from django.utils.translation import gettext_lazy as _
+from tools.models import Organization
+from tools.view_helpers import StepsMixin
+
+from reports.views.base import get_selection
+
+
+class KatGenerateReportStepsMixin(StepsMixin):
```

Comment:
```suggestion
class KATGenerateReportStepsMixin(StepsMixin):
```

Reply:
The correct form would be the `KAT` prefix, since it's an abbreviation. It would be nicer to change the names in the current PR, the change should be pretty straightforward without much risks.

Reply:
I think we can completely remove Kat or KAT

Reply:
> I think we can completely remove Kat or KAT

That would even be better, the prefixes don't add much within this context

Reply:
I just updated it and removed "KAT" from the class names for both reports and onboarding.